FaqGen/tests/test_faqgen_on_gaudi.sh

```diff
@@ -0,0 +1,163 @@
+#!/bin/bash
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+set -x
+
+WORKPATH=$(dirname "$PWD")
+LOG_PATH="$WORKPATH/tests"
+ip_address=$(hostname -I | awk '{print $1}')
+
+function build_docker_images() {
+    cd $WORKPATH
+
+    git clone https://github.com/opea-project/GenAIComps.git
+    cd GenAIComps
+    docker build --no-cache -t opea/llm-faqgen-tgi:latest --build-arg https_proxy=$https_proxy --build-arg http_proxy=$http_proxy -f comps/llms/faq-generation/tgi/Dockerfile .
+
+    docker pull ghcr.io/huggingface/tgi-gaudi:1.2.1
```

Comment:
The habana driver version of the test machine is v1.16.1, the tgi-gaudi should be ghcr.io/huggingface/tgi-gaudi:2.0.1, I think that may be the issue why test failed.

Reply:
Artifact download URL: https://github.com/opea-project/GenAIExamples/actions/runs/10057922170/artifacts/1730592345

Reply:
Hi @chensuyue , thanks updated the tgi-gaudi  Version
All tests are passed now